daemon/lib/source/DobbyManager.cpp

```diff
@@ -2259,6 +2259,8 @@ void DobbyManager::handleContainerTerminate(const ContainerId &id, const std::un
         // Also run any postHalt hooks in RDK plugins
         if (container->config->rdkPlugins().size() > 0)
         {
+            AI_LOG_INFO("Exit status = %d",status);
```

Comment:
Could you also add here the container ID (preferable string representation) so we would know which container exit status is that.